database/migrations/2018_07_16_180920_create_addresses_table.php

```diff
@@ -21,6 +21,7 @@ public function up()
             $table->string('state');
             $table->string('postal_code');
             $table->string('country');
+            $table->unsignedInteger('user_id');
```

Comment:
This going to break other tests which doesn't expect to set `user_id` when testing address.